src/index.js

```diff
@@ -4,7 +4,7 @@
  * @copyright (c) 2017 Kent C. Dodds
  * @author Kent C. Dodds <kent@doddsfamily.us>
  */
-import diacritics from 'diacritic'
+import removeAccents from 'remove-accents'
```

Comment:
Comparing the implementation here, I think there are some things that `diacritic` supports which `remove-accents` does not support:

https://unpkg.com/diacritic@0.0.2/diacritics.js vs https://unpkg.com/remove-accents@0.4.2/index.js

Am I missing something? I feel like this could be a breaking change/regression for people.

Reply:
Absolutely yes, diacritics.js has much more diacritic symbols, but as I noticed one of them is definitely not correct, probably there are others.

Also neither of packages I found seem to support removing [combining characters](https://en.wikipedia.org/wiki/Combining_character) or unicode normalization.

Are you willing to use string.normalize?
https://developer.mozilla.org/ru/docs/Web/JavaScript/Reference/Global_Objects/String/normalize
(should consider browser support / requiring a 130kb [polyfill](https://github.com/walling/unorm/blob/master/src/unormdata.js)  )

So with the exception of making a new js diacritic library (probably by pulling [data from wikipedia](https://en.wikipedia.org/wiki/List_of_Unicode_characters) and adding string.normalize() I belive this to be the best option.

Reply:
Ok, I'm ok with this change. I'd rather not require people support IE to use a polyfill.

I am going to release this as a breaking change though. Thanks!